### DISCUSSION TITLE
TypeError in the examples

### DISCUSSION THREAD

**Discussion by Feargus** [pymeasure/pymeasure#624] [Q&A] [answered]
Running the gui.py example, I get the following error when pressing the queue button.

```
 in send_multipart    memoryview(msg)
TypeError: memoryview: a bytes-like object is required, not 'int'
[...]
During handling of the above exception, another exception occurred:
[...]
"Frame 0 (128) does not support the buffer interface
```

I am using python 3.8.8 (on a fresh install of Anaconda, Windows), and I am running the example by calling `python gui.py`

**Comment by bilderbuchi** [comment, marked as answer]
Is this with the latest release, pymeasure 0.10?

**Reply by Feargus** [reply]
It was 0.9. I was so focused on the python version, that the package completely slipped my mind. Thanks and sorry for the trouble

**Reply by bilderbuchi** [reply]
No problem :-)
